app/controllers/line_items_controller.rb

```diff
@@ -1,7 +1,10 @@
 class LineItemsController < BaseController
   respond_to :json
 
-  def index
+  # Taken from Spree::Api::BaseController
+  rescue_from ActiveRecord::RecordNotFound, :with => :not_found
```

Comment:
a `before_filter` would be more explicit IMHO, you probably don't want to rescue all the AR::RNF but only known one

https://github.com/openfoodfoundation/openfoodnetwork/pull/1536#discussion_r116174434